src/models/launch_1/customization.md

```diff
@@ -0,0 +1,95 @@
+# Launch (Usage & Customization)
+
+The Launch Configurable Keyboard can be customized and personalized in a variety of ways. It is recommended to unplug all USB cables and devices from your Launch before changing keycaps or switches.
+
+- [Connecting and Using Launch](#connecting-and-using-launch)
+- [Removing and Installing Keycaps](#removing-and-installing-keycaps)
+- [Removing and Installing Switches](#removing-and-installing-switches)
+- [Removing the PCB](#removing-the-pcb)
+
+## Connecting and Using Launch
+
+![Launch ports](./img/launch-ports.png)
```

Comment:
This image highlights a number of downstream ports that would not work for connecting to the computer. Can we get a photo with only the port we're talking about highlighted? (I would take a head-on photo and highlight with a box in Inkscape, but if we really want to reuse the port diagram photo, just removing the rest of the labels would also work.)

Reply:
You could also highlight them all with boxes, but make the center one a different color. That way, you can more clearly point out which one to use for the computer and which ones can be used for downstream devices.